kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/QuizResourceSelection/subPages/QuestionsSettings.vue

```diff
@@ -4,6 +4,12 @@
     <div class="mb-20">
       {{ maxNumberOfQuestionsInfo$({ count: maxQuestions }) }}
     </div>
+    <UiAlert
+      v-if="addableQuestionCount < 25"
+      type="warning"
```

Comment:
Since `dismissible` is true by default, leaving it out of the props here makes sense. But, it doesn't seem like the dismissing is actually working. Can you take a look into that, and see if there's something that needs to be changed on the kolibri side here (maybe something with the DOM updating - or not updating - properly) or if there might be an issue with UiAlert? If it seems to be a KDS problem, we can evaluate priority, and it might not be a blocker for this PR.

Reply:
I got it now , we needed to add an event listener.